### ISSUE TITLE
License stuff

### ISSUE THREAD

**Issue by ca5ua1** [muelphil/smartclip#3]
Well, i here to chuckle a little and point to why I and folks that enjoy Free Software won't use your program.

LICENSE:
```
3. Restrictions
You agree not to:
...
Decompile, reverse-engineer, disassemble, or otherwise attempt to derive the source code of Smartclip.
```
_Bro, don't sue me for downloading and/or viewing source code of Smartclip© from your GitHub™ account XD_

---

And about part that why I won't use your program - your license prohibits viewing source code of your program but it also prevents me to help you improve your by fixing any issues that program may have and/or add own features. 

LICENSE:
```
3. Restrictions
You agree not to:
Modify, alter, adapt, translate, or create derivative works of Smartclip.
Decompile, reverse-engineer, disassemble, or otherwise attempt to derive the source code of Smartclip.
```

So essentially what license says, that you have proprietary program but that entirely don't corelate that with source code available. I just hope you've picked template to prohibit commercial use without mentioning this issue.

I understand that You probably wanted protect your work, but having proprietary license just scare off developers that may be interested in your product and some ~insane~ users that read licenses and care about them (majority of OSI and FSF supporters)

**Comment by muelphil** [comment]
I want to respond to some of your points:
1. Please speak for yourself. While I respect your response and your opinion, you cant speak for many, let alone all "folks that enjoy Free Software". I have many friends in the computer science field that enjoy free and open software and still use my Software. While they prefer open software, it doesn't mean they wont use proprietary software at all. I know there are subgroups that only use software thats open source, if you are one of them, move on, this is not for you.
2. Claiming a piece of software is only worth something when you can alter it, is an interesting opinion. Many of the Software Development Tools are proprietary, Operating Systems are proprietary, Drivers are proprietary and you still use all of those too.
3. I value your offer for helping me, but I think I'm good without you. Issuing bugs is helpful, but I can very well fix them on my own, I don't exactly need help doing so by someone who has written a few lines of Python. Smartclip is a big project with hundreds of files of sourcecode, several operating system specific native implementations and an expandable architecture, that I manage on my own. This isnt a half baked unfinished product, there may be bugs that I haven't spotted yet, but I didnt put it on here in an effort to get other developers to finish my work, thank you very much.
4. This is my private hobby and I spend months on end working on it in my freetime. I put it out there to help people optimize their workflow, after I originally just worked on it for myself. I dont make any money with it and its a helpful tool you can use for free. I have no understanding for people who cant respect the work of others.
5. The source code of Smartclip is not available, which you would have seen if you actually looked inside the repo. This is just the documentation, which I also use to post releases.
6. Where is the source for "license scaring off developers that are interested in my product"? Again, speak for yourself.
7. This is a normal proprietary license, noone is going to sue you for downloading and using the application. Again, you wont be able to read the source code without decompiling it, which is near impossible at the moment anyway. This is a security measure to protect my work from being used by companies for their own products, that they sell for money.
8. Depending on the response to it, I want to make Plugin-development for Smartclip available. This will allow interested developers to extend the functionality of Smartclip. As I'm doing this in my free time and I don't know how many people are interested in using it, I havent found the time or energy to implement that, as that would require setting up a reasonable development environment to implement and test plugins.

You made it reasonable clear that you are not interested in proprietary software, I do respect that, but would ask you to just move on in this case, Smartclip isn't for you then.

**Comment by ca5ua1** [comment]
1. Yep, you are right. When I wrote "folks that enjoy Free Software" I actually meant enthusiast that run gentoo/freebsd/etc.. without using any proprietary software.
2. I didn't claimed that. And "you still use all of those too" - I disagree with that, apart from entertainment, the only proprietary software that I use are drivers in linux kernel + GPU/Motherboard BIOSes + CPU microcode.
3. "I don't exactly need help doing so by someone who has written a few lines of Python" pretty toxic. I don't use GitHub, I have my own Git server.
4. "I have no understanding for people who cant respect the work of others." - I respect your work, the entire issue are made from my misunderstanding - see #5. 
5. Oh, I see, I thought this repo contains source code of your app. I've checked repo a bit and after seeing Ruby gems I thought it contains source code. I've found it funny that you have access to source code and icense says that it's illegal to have access to source code and wanted to tell you about it with downsides of this license. So, my bad for all this misunderstanding. 
6. See #5,1
7. See #5
8. Plugin systems always great. In my opinion your userbase isn't big enough that there would be someone greatly interested in this feature, thou I can be wrong. 

Yeah, in the end it's my bad that I've didn't looked in repo more throughtfully to find out that there is no source code. Sorry for consuming both of our time on this. I think you was a bit emotional that lead to your big response instead of simply writting something like _"Bro, you are wrong, source code isn't available. My program is proprietary."_
